scylla/src/frame/request/batch.rs

```diff
@@ -0,0 +1,98 @@
+use anyhow::Result;
+use bytes::{BufMut, Bytes};
+
+use crate::frame::{
+    request::{Request, RequestOpcode},
+    types,
+    value::Value,
+};
+
+pub struct Batch<'a, 'b, I: Iterator<Item = BatchStatementWithValues<'a, 'b>> + Clone> {
+    pub statements: I,
+    pub num_of_statements: usize,
+    pub batch_type: BatchType,
+    pub consistency: i16,
+}
```

Comment:
I don't think there is a need for two separate lifetimes. Please use only `'a`.

Reply:
Done